core/version.go

```diff
@@ -2,5 +2,10 @@ package core
 
 import _ "embed"
 
+const (
+	CheckVersionEndpoint = "https://download-page.deref.workers.dev/version.txt"
+	UpdateScriptEndpoint = "https://download-page.deref.workers.dev/install.sh"
```

Comment:
We don't control `workers.dev`, so let's use the redirecting endpoint to insulate ourselves from the implementation service.